SoObjects/Appointments/SOGoCalendarComponent.m

```diff
@@ -130,6 +130,25 @@ + (SOGoWebDAVAclManager *) webdavAclManager
   return aclManager;
 }
 
+- (void) setIsNew: (BOOL) newIsNew
+{
+  // Required for protocol <SOGoComponentOccurence>
+  // but not yet implemented.  Violently warn if
+  // used.
+  NSLog (@"SOGoCalendarComponent::setIsNew is not implemented.");
+  abort();
+}
+
+- (BOOL) isNew
+{
+  // Required for protocol <SOGoComponentOccurence>
+  // but not yet implemented.  Violently warn if
+  // used.
+  NSLog (@"SOGoCalendarComponent::isNew is not implemented.");
+  abort();
+  return false;
```

Comment:
`isNew` is actually implemented by the parent class `SOGoContentObject`, so simply return this:

```objc
    return [super isNew];
```